rs_prod.ini

```diff
@@ -5,17 +5,14 @@ s3_upload=false
 
 # Enable rs upload
 remote_settings_upload=true
-remote_settings_url=%(SHAVAR_REMOTE_SETTINGS_URL)s
+remote_settings_url=%(SERVER)s
 # The remote settings bucket that has the tracking-protection-lists collection
 remote_settings_bucket=main-workspace
 # The remote settings collection that has the record of each safebrowsing formatted list
 remote_settings_collection=tracking-protection-lists
-# Auth info required if remote_settings_upload is true
-remote_settings_username=%(SHAVAR_REMOTE_SETTINGS_USERNAME)s
-remote_settings_password=%(SHAVAR_REMOTE_SETTINGS_PASSWORD)s
-
-# Token auth
-remote_settings_token=%(SHAVAR_REMOTE_SETTINGS_TOKEN)s
+# Credentials for building the Authorization Header
+# (eg. Bearer f8435u30596, Basic some-user:some-password)
```

Comment:
```suggestion
# (eg. "Bearer f8435u30596", or for basic auth "some-user:some-password")
```

(I know you took this from our docs and I'm sorry 🙏

Reply:
Fixed!